lib/main_common.pm

```diff
@@ -2426,4 +2426,23 @@ sub load_transactional_role_tests {
     loadtest 'transactional_system/rebootmgr';
 }
 
+sub load_lvm_tests {
```

Comment:
Could you, please, add couple of comments and ideally describe variables.

Reply:
Added.